main/src/ca/uwaterloo/flix/language/phase/Statistics.scala

```diff
@@ -156,6 +156,7 @@ object Statistics {
       case Expression.FixpointFilter(pred, exp, tpe, pur, eff, loc) => visitExp(exp)
       case Expression.FixpointInject(exp, pred, tpe, pur, eff, loc) => visitExp(exp)
       case Expression.FixpointProject(pred, exp, tpe, pur, eff, loc) => visitExp(exp)
+      case Expression.Instanceof(exp, cls, loc) => visitExp(exp)
```

Comment:
cls -> _

(We should be better at using _ ourselves).

Reply:
Done